### ISSUE TITLE
[checkbox] checkbox cant change color

### ISSUE THREAD

**Issue by devopg** [fomantic/Fomantic-UI#2225]
```
  <div class="ui right aligned toggle checkbox">
        <input type="checkbox" checked="checked">
        <label>Toggle</label>
      </div>
```
always blue:
![image](https://user-images.githubusercontent.com/61819835/152569021-d70b6b82-2549-4d85-b3a5-ff74c986e978.png)

because you crazy?! :D 
plz remove important 

![image](https://user-images.githubusercontent.com/61819835/152569103-2d6963f5-9757-47e8-a7f9-b720f5255755.png)

**Comment by lubber-de** [comment]
That is currently in progress by #2010. I mentioned it here https://github.com/fomantic/Fomantic-UI/pull/2010#discussion_r684601281

In the meantime you can of course override the color by adding `!important` to your own class override. Yes, it's bad practice, but working.

**Comment by devopg** [comment]
no add !important dont work because how you know so big rule chain has more priority. only if add id="checkbox" and set color to it id

**Comment by lubber-de** [comment]
Fixed by #2686
